js/templates/modals/orderDetail/orderDetail.html

```diff
@@ -4,11 +4,10 @@
     // and componentize the css.
   %>
   <% if (ob.returnText) { %>
-  <div class="contentBox clrP clrSh3 clrBr clrT">
-    <a class="padSm gutterHSm overflowAuto margRSm flexVCent returnBox js-returnBox">
-      <span class="clrBr2 clrSh1 disc avatar flexNoShrink" style="<%= ob.getAvatarBgImage(ob.ownProfile.avatarHashes) %>"></span>
-      <span class="txUnl tx3 clamp clrT name"><%= ob.ownProfile.name %></span>
-      <span class="txU flexNoShrink tx6"><%= ob.returnText %></span>
+  <div class="btnStrip clrSh3">
```

Comment:
Since the `returnBox` styling is no longer needed, can you remove it from `_orderDetail.scss`?

Also, with what was taken out above, this template no longer needs `ownProfile`, can you remove the following last line from `OrderDetail.js`?

```
  render() {
    loadTemplate('modals/orderDetail/orderDetail.html', t => {
      const state = this.getState();

      this.$el.html(t({
        ...state,
        ...this.model.toJSON(),
        ownProfile: app.profile.toJSON(), // remove this line since ownProfile is no longer needed in the template
```